app/javascript/psulib_blacklight/javascripts/google_preview.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(() => {
+  const element = document.getElementById('google-preview');
+  const googlePreviewUrl = '/preview/google-preview-data';
+  const googlePreviewUrlQuery = `${googlePreviewUrl}/?search_item=${element.attributes.data.value.toString()}`;
+
+  if (element) {
+    $.ajax({
+      url: googlePreviewUrlQuery,
+      success(response) {
+        for (const key in response) {
```

Comment:
Does this need to be a loop? Do we think there may potentially be more than one response here? Or that the response key may not match up exactly with the `search_item` param we are passing in?

I'm fine if the answer to any of these questions is yes; just want to make sure I'm understanding the rationale.

I was thinking we could just do

```
if (response[search_item])
  //do stuff
```

but maybe that isn't reliable enough and your approach would catch the scenarios above.